mobile/mobile_46.asm

```diff
@@ -1484,7 +1484,8 @@ Function118b9a:
 	and a
 	ret
 
-	charmap " ", $20 ; revert to ascii
+	pushc
+	setcharmap ascii ; revert to ascii
```

Comment:
This use of setcharmap should end after the URLs.

Reply:
Resolved in latest commit.